File: back/wildcircus/src/main/java/com/checkpoint/wcs/wildcircus/entities/Ticket.java
package com.checkpoint.wcs.wildcircus.entities;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Ticket {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String category;
    private String period;
    private Float price;

    public Ticket() { }

    public Ticket(String category, String period, Float price) {
        this.category = category;
        this.period = period;
        this.price = price;
    }

    public Long getId() {
        return id;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getPeriod() {
        return period;
    }

    public void setPeriod(String period) {
        this.period = period;
    }

    public Float getPrice() {
        return price;
    }

    public void setPrice(Float price) {
        this.price = price;
    }
}

File: front/src/app/models/ticket.ts
export class Ticket {

    private id?: number;
    private category: string;
    private period: string;
    private price: number;

    constructor(category: string, period: string, price: number, id?: number) {
        this.id = id;
        this.category = category;
        this.period = period;
        this.price = price;
    }

    public getId() {
        return this.id;
    }

    public getCategory() {
        return this.category;
    }

    public getPeriod() {
        return this.period;
    }

    public getPrice() {
        return this.price;
    }
}
